test/libsolidity/syntaxTests/functionTypes/valid_function_type_variables.sol

```diff
@@ -18,7 +18,6 @@ contract test {
     function(uint) pure internal public h = fh;
 }
 // ----
-// Warning: No visibility specified. Defaulting to "public".
```

Comment:
This was removed because the check moved from the type checker to the static analyzer. The static analyzer is not run if the type checker reported an error.